docs/concepts/multiaddr.md

```diff
@@ -0,0 +1,40 @@
+---
+title: 'Multi-addresses'
+description: Multi-addresses and how they are used in IPFS
+---
+
+# Understanding Multi-Addresses in IPFS
+
+A [multiaddr](https://github.com/multiformats/multiaddr) encodes multiple layers of addressing information into a single "future-proof" path structure.
+
+It is a self-describing format that allows multiple protocols encoded in a single string. This allows for flexibility in addressing and routing network traffic in a decentralized system like IPFS.
+
+A multiaddr has the following format:
+
+```shell
+/<protocol>/<address>
+```
+
+Where `<protocol>` is the name of the network protocol being used, and `<address>` is the address of the node on that network.
+
+For example, a multiaddr for an IPFS node with the IP address `192.0.2.1` and listening on port `5001` would be:
```

Comment:
You describe the mutliaddr example twice, so I'd suggest deleting the note (`>`) and consolidating

```
For example, this multiaddr indicates that the node uses the IPv4 protocol, has the IP address 192.0.2.1, and listens on TCP port 5001.
```